src/Orleans.Connections.Security/Security/TlsConnectionAdapterOptions.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Net.Security;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using Microsoft.AspNetCore.Connections;
+
+namespace Orleans.Connections.Security
+{
+    /// <summary>
+    /// Settings for how TLS connections are handled.
+    /// </summary>
+    public class TlsConnectionAdapterOptions
```

Comment:
Don't replicate the Adapter naming mistake we made 😄

Reply:
s/Https/Tls 😅 
Perhaps `TlsOptions` is a nicer name for this, or `TlsConnectionOptions`